libs/etterlatte-brev-model/src/main/kotlin/no/nav/etterlatte/brev/EtterlatteBrevkode.kt

```diff
@@ -40,6 +40,7 @@ enum class EtterlatteBrevKode(
     OMSTILLINGSSTOENAD_VEDLEGG_FORHAANDSVARSEL_UTFALL(Brevtype.VEDLEGG, "Utfall ved forhåndsvarsel av feilutbetaling"),
     OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_VARSELBREV_INNHOLD(Brevtype.MANUELT, "Varsel om aktivitetsplikt for omstillingsstønad"),
     OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_INFORMASJON_4MND_INNHOLD(Brevtype.MANUELT, "Informasjon om aktivitetsplikt for omstillingsstønad"),
+    OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_INFORMASJON_6MND_INNHOLD(Brevtype.MANUELT, "Informasjon om aktivitetsplikt for omstillingsstønad"),
```

Comment:
Er det noen klare retningslinjer for når man skal bruke `MANUELT` og `INFORMASJON`?